docs/proposals/ose-rebase.md

```diff
@@ -0,0 +1,39 @@
+## Rebase OSE on Origin
+
+Today, code in OSE lands by fetching and merging Origin manually. OSE-specific changes
+are scattered in the repo and it's hard to reason about any conflicts. This doc proposes
+moving all the changes in OSE that diverge from Origin into a set of commits that
+will sit on top of the Origin code[0] and a Jenkins job that will be rebasing master and
+all active release branches on a daily basis.
+
+There are two types of commits that need to be carried on top of Origin: 
+1. long-term carries related to branding and release tooling (master and release branches)
+2. CVEs (release branches only)
```

Comment:
Technically there is another type: bug-fixes that are not CVEs but that we do not back-port to Origin releases (since we do not want to incur the maintenance cost).